website/registrations/views.py

```diff
@@ -190,6 +190,47 @@ def post(self, request, *args, **kwargs):
         return super().post(request, *args, **kwargs)
 
 
+class NewYearRenewalFormView(FormView):
```

Comment:
This would be a good place for a docstring explaining what it does. Additionally, the registrations app readme can use an update